core/controlplane/config/config.go

```diff
@@ -284,6 +284,10 @@ func (c *Cluster) SetDefaults() {
 			c.Etcd.Subnets = c.PublicSubnets()
 		}
 	}
+
+	if c.Etcd.InternalDomainName == "" {
+		c.Etcd.InternalDomainName = fmt.Sprintf("%s.internal", c.ClusterName)
```

Comment:
This is now dead-code which had been used for the ENI+Route53 Record Set way of implementing the network identity for etcd nodes.
I can revive it if you really need it rather than the current, EIP way.